src/meta/Intersection.ts

```diff
@@ -54,8 +54,11 @@ export class Intersection extends Base {
 		}
 
 		let details = this._getDetails(options);
-		if (!details) {
-			details = this._createDetails(options, rootNode);
+		if (!details || !details.entries.get(node)) {
+			if (!details) {
```

Comment:
am i missing a subtlety here? (presuming if there isn't any details that there won't be an entry?), could the code not just be:

```typescript
details = details || this._createDetails(options, rootNode)
if (!details.entries.get(node)) {
    details.entries.set(node, defaultIntersection);
    details.observer.observe(node);
}
```

Reply:
Yeah no reason not to simplify it that way. Code's been updated